pysmt/solvers/solver.py

```diff
@@ -241,6 +240,15 @@ def set_options(self, options):
         :type options: Dictionary
         """
         raise NotImplementedError
+    
+    def set_option(self, name, value):
```

Comment:
This should not be here, please remove it. Currently, we do not have a uniform way of passing options to the solver *after* it was created. This needs much more discussion to identify a way of doing it in a uniform way among all solvers.